composer.json

```diff
@@ -15,9 +16,9 @@
         }
     },
     "require": {
-        "php": ">=5.4.0",
-        "gusdecool/cryptography": "~1.0",
-        "doctrine/orm": "~2.2",
-        "sensio/framework-extra-bundle": "~2.3|~3.0"
+        "php": "^7.1.3",
+        "gusdecool/cryptography": "~1.1|dev-mcrypt_to_openssl",
```

Comment:
remove `dev-mcrypt_to_openssl`, we should not using `branch` for production usage

Reply:
you can change this to `~2.0` ✌️